jdi-bdd-tests/src/test/java/cucmberTests/TestRunner.java

```diff
@@ -22,11 +21,12 @@
 @CucumberOptions(
         features = "classpath:features"
         , glue = {"com.epam.jdi.bdd", "cucmberTests"}
-        //, tags = {"@webpage"}
+        // , tags = {"@radio"}
 )
-public class TestRunner extends AbstractTestNGCucumberTests {
+public class TestRunner {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestRunner' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524853)